source-control/readme.md

```diff
@@ -32,11 +32,63 @@ The team should at least be doing the following:
 
 ## Commit Best Practices
 
-TODO: Add more content and details here!
+* Make small commits. This makes changes easier to review, and if we need to revert a commit, we loose less work.
```

Comment:
`loose` -> `lose`

Reply:
Apologies, lots of typos in here, wish I had a typo extension in VS Code :) will fix them all

Reply:
fixed